### DISCUSSION TITLE
Kağıtlarla oynanan mantık oyunu

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#111] [Zeka Sorusu]
Mükemmel mantığa sahip Ayşegül ile Göksel bir oyun oynarlar. 24 tane kağıda 1 den 6 ya kadar sayılar yazılır (her sayı 4 kere) ve kağıtların numaralandırılmış yüzleri yukarı bakacak şekilde yere koyulur. Bundan sonra Ayşegül’den başlamak üzere sırayla her biri o ana kadar çevrilmemiş herhangi bir kağıdı ters çevirirler. Yaptığı hamleyle ters çevrilmiş kağıtların toplamını 30 u geçirenin (31-32-33-34-35-36 yapanın) kaybettiği bu oyunu hangisi kazanır?

**Comment by Ersan0102** [comment]
30 puana ulaşmak istemedikleri için sayı değeri en düşük olan kartlardan çevirmeye başlarlar. Üzerinde beş yazan kağıtlara kadar puanları eşittir. Fakat üzerinde 6 yazan kartlara geldiklerinde sıra oyuna ilk başlayan Ayşegül'de olduğu için Ayşegül üzerinde 6 yazan ilk kartı ters çevirir ve 36 puana ulaşıp oyunu kaybeder. Oyunu Göksel kazanır.
